plane/source/docs/automatic-takeoff.rst

```diff
@@ -26,8 +24,8 @@ for a wide range of aircraft.
 
 During takeoff the wings will be held level to within
 :ref:`LEVEL_ROLL_LIMIT <LEVEL_ROLL_LIMIT>`
-degrees until 5 meters of altitude is reached, gradually allowing more roll until
-the normal :ref:`ROLL_LIMIT_DEG<ROLL_LIMIT_DEG>` limit is allowed once 15 meters of altitude is
+degrees until :ref:`TKOFF_LVL_ALT<TKOFF_LVL_ALT>` (10m by default) meters of altitude is reached, gradually allowing more roll until
+the normal :ref:`ROLL_LIMIT_DEG<ROLL_LIMIT_DEG>` limit is allowed once 3x :ref:`TKOFF_LVL_ALT<TKOFF_LVL_ALT>` meters of altitude is
```

Comment:
We increased that limit from 5 to 10 as per your request recently @Hwurzburg . Seeing how it causes limited roll up until 3 times that, do you think we should consider reverting to 5m?